### ISSUE TITLE
Windows 10 labels latest version as Trojan

### ISSUE THREAD

**Issue by ntjohnst** [Razzmatazzz/RemnantSaveManager#41]
When trying to run the latest version windows 10 warns the user and when run anyway stops the program and deletes it.

![Captureone](https://user-images.githubusercontent.com/6242372/94349193-6a6df580-0010-11eb-8b0b-844d72add28d.PNG)
![Capturetwo](https://user-images.githubusercontent.com/6242372/94349194-6b068c00-0010-11eb-96d5-3297cd7828c4.PNG)

**Comment by GrandNegus68** [comment]
Had the same issue, looking at code branch this must be a false positive

**Comment by Razzmatazzz** [comment]
Yes, [the false positive has been an issue for a while](https://github.com/Razzmatazzz/RemnantSaveManager/issues/31). Unfortunately, not much I can do about it.

**Comment by devious21sf** [comment]
This behavior at least for me, is new as of this version. Defender wasn't attempting to quarantine on launch before this version.

**Comment by Razzmatazzz** [comment]
It might be new to you, but I can't control when/if Windows Defender detects anything as a virus.

**Comment by ntjohnst** [comment]
Since there is nothing to be done about this marking as closed.

**Comment by Razzmatazzz** [comment]
Let's leave this open since other folks are likely to open similar issues in the future. To anyone who comes here in the future: it's a known issue that Windows Defender reports the app as a trojan. There isn't anything I can do to stop the way Windows Defender behaves. If you don't trust the pre-compiled release, feel free to download the source code and compile it yourself.

**Comment by devious21sf** [comment]
I submitted the zip to https://www.microsoft.com/en-us/wdsi/filesubmission for 1.95.40. 

Result here (took about an hour): https://www.microsoft.com/en-us/wdsi/submission/d7b2aeaa-325f-42d1-a769-e0bf0ced0dfd
Just tested and confirmed it's no longer being detected.

**Comment by ntjohnst** [comment]
Seems to work fine now, thanks devious.

**Comment by GrandNegus68** [comment]
Thanks Devious / Razzmatazzz, all good here after updating virus security "intelligence" to 1.323.2132.0, good work !

**Comment by HandyBobNeedsHelpToo** [comment]
I ran into this issue as well, when I thought it was finally fixed I kept getting a message saying Microsoft Defender SmartScreen prevented an unrecognized app from starting. Any idea how to fix this so I can finally start using Save Manager?
![cantopen](https://user-images.githubusercontent.com/107770090/175889683-84a88ad0-cf9d-4e39-aa76-53a39d2f8aa6.png)

**Comment by Razzmatazzz** [comment]
I reported it as a false positive at the link devious21sf provided. I don't know if that will help, but it's all I can do.
